rc/filetype/fennel.kak

```diff
@@ -0,0 +1,113 @@
+# http://fennel-lang.org
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .*[.]fnl %{
+    set-option buffer filetype fennel
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+hook global WinSetOption filetype=fennel %{
+    require-module fennel
+    fennel-configure-window
+}
+
+hook -group fennel-highlight global WinSetOption filetype=fennel %{
+    add-highlighter window/fennel ref fennel
+    hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/fennel }
+}
+
+hook -group fennel-insert global BufNewFile .*[.]fnl %{
+    require-module fennel
+}
```

Comment:
I dont quite understand the point of this hook, its unusual to just require a filetype module without using something from it.

Reply:
oh, perhaps a copy mistake. Clojure.kak did something there, but I removed everything not related to fennel and forgot to clean it up

Reply:
@mawww, should be fine now